### PR TITLE
refactor: remove legacy subcommand syntax

### PR DESCRIPTION
**Pull request by miurahr**
- Drop CI test for legacy commands
- Drop test cases for legacy commands
- Drop logics for legacy commands

### CODE REVIEW

**Comment by miurahr** [comment]
Call for code reviewer.

File: aqt/installer.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 #
 # Copyright (C) 2018 Linus Jahn <lnj@kaidan.im>
-# Copyright (C) 2019-2021 Hiroshi Miura <miurahr@linux.com>
+# Copyright (C) 2019-2021,2024 Hiroshi Miura <miurahr@linux.com>
```
Comment (pzhlkj6612) [inline aqt/installer.py:4]: 2022 & 2023?

Comment (miurahr) [inline aqt/installer.py:4]:
```suggestion
# Copyright (C) 2019-2024 Hiroshi Miura <miurahr@linux.com>
```

File: tests/test_cli.py [hunk 2]
```diff
@@ -110,16 +105,12 @@ def test_cli_invalid_version(capsys, invalid_version):
     cli._setup_settings()
 
     matcher = re.compile(
-        r"^INFO    : aqtinstall\(aqt\) v.* on Python 3.*\n"
+        #  r"^INFO    : aqtinstall\(aqt\) v.* on Python 3.*\n"
```
Comment (pzhlkj6612) [inline tests/test_cli.py:108]: Why commented out not removed?

Comment (miurahr) [reply tests/test_cli.py:108]: Oh, I've forgotten to restore the version message.

**Pull request merged**